src/lib/actions/create-activity.js

```diff
@@ -0,0 +1,34 @@
+const { isValidName } = require('../tools/validate');
+const { errors } = require('../../constants');
+
+function createActivity(params, activities) {
+  /* check request parameters */
+  if (typeof params.name !== 'string' || params.name.length < 1 || params.name.length > 80) {
+    throw new Error(`${errors.common.INVALID_PARAMETER_VALUE}: --name`);
+  }
+  if (!isValidName(params.name)) {
+    throw new Error(errors.createActivity.INVALID_NAME);
+  }
+  const match = activities.find(a => a.name === params.name);
+  if (match) {
+    // NOTE: No error specified for this case
```

Comment:
you mean not in the doc? we should try it on AWS and see if this error is thrown or not

Reply:
Yes, will check on AWS.

Reply:
Checked. If you create an activity with the same name, it returns the first activity created, without error.
> Will update the code